types/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
I think we should have a single `CHANGELOG` or things will become messy and hard to read and keep up to date.

Reply:
yeah that would be good but we don't have a wrapper/facade crate yet... :) 

I can remove the CHANGELOGs for now